README.md

```diff
@@ -66,31 +67,44 @@ Before installing deeprankcore you need to install some dependencies. We advise
   - Here, `${TORCH}` and `${CUDA}` should be replaced by the pytorch and CUDA versions installed. You can find these using:
     - `python -c "import torch; print(torch.__version__)"` and
     - `python -c "import torch; print(torch.version.cuda)"`
+      - if this returns `None`, use `cpu` instead
   - For example: `https://data.pyg.org/whl/torch-2.0.0+cpu.html`
-* Only if you have a MacOS with M1 chip, run also:
-  * `conda install pytables`
+* Check if gcc is installed: `gcc --version`.
```

Comment:
For which dependency is this needed? I had it already on my mac so I am not sure

Reply:
I think for freesasa, if I remember correctly

Reply:
indeed I also think it's not necessary